android/app/src/main/java/com/bounswe2020group3/paperlayer/project/MembersAdapter.kt

```diff
@@ -45,12 +47,19 @@ class MembersAdapter(val clickListener: OnMemberCardClickListener): RecyclerView
     ): RecyclerView.ViewHolder(itemView) {
 
         var memberUsername: TextView =itemView.textViewProjectMemberUsername
+        var memberAvatar: ImageView = itemView.imageViewMemberAvatar
 
         fun bind(memberCard: MemberCard, clickListener: OnMemberCardClickListener){
-            memberUsername.setText(memberCard.username)
+            val fullName = "${memberCard.user.profile[0].name} ${memberCard.user.profile[0].lastName}"
+            memberUsername.text = fullName
+
+            val imageUrl = memberCard.user.profile[0].profilePicture
+            if (imageUrl != null && imageUrl != "") {
```

Comment:
For easier code writing you can use isNullOrEmpty() function on strings. This is not related to pull request, just a tip.